Makefile

```diff
@@ -23,44 +24,48 @@ include build-infra/cw-deb.mk
 include build-infra/python.mk
 
 .PHONY: fvtest
-fvtest: fvtest_setup.py env ${ENV_DIR}/.test_requirements
+fvtest: fvtest_setup.py env ${ENV_DIR}/.test-requirements
 	PYTHONPATH=src:common ${PYTHON} fvtest_setup.py test -v
 
-.PHONY: test
-test: coverage
-
-coverage: ${ENV_DIR}/.test_requirements
-
 .PHONY: test_cluster_mgr
-test_cluster_mgr: cluster_mgr_setup.py env ${ENV_DIR}/.test_requirements
+test_cluster_mgr: cluster_mgr_setup.py env ${ENV_DIR}/.test-requirements
```

Comment:
Should we commonalise these three blocks into a macro - e.g. as part of `etcd_component`?